src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -183,6 +183,8 @@ private async Task ImportArchive(MigrateRepoCommandArgs args, string archiveUrl
 
         archiveUrl ??= args.ArchiveUrl;
 
+        var bbsRepoUrl = GetBbsRepoUrl(args);
```

Comment:
Can we add a few tests for the behavior of `GetBbsRepoUrl`?

Reply:
The `Happy Path Ingest` tests should cover this? 

https://github.com/github/gh-gei/pull/1002/files#diff-a49ae4d3e39f49629ff7581c6766e5941b5d26dc5fea7878432ba71a3da0c04eR123

Reply:
We're also planning on making the bbs url, project, and repo args required in followup issue https://github.com/github/gh-gei/issues/1004.